messages/en.json

```diff
@@ -60,7 +63,9 @@
         "Despite protests, the mob drags their victim to the hanging tree. \u0002{0}\u0002 succumbs to the will of the horde, and is hanged.",
         "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows.",
         "Before the rope is pulled, \u0002{0}\u0002 pulls the pin on a grenade. They hesitate, and it explodes, killing them.",
-        "Before the rope is pulled, \u0002{0}\u0002 throws a grenade at the mob. The grenade explodes early."
+        "Before the rope is pulled, \u0002{0}\u0002 throws a grenade at the mob. The grenade explodes early.",
+        "Uttering a chilling war cry, the villagers drag \u0002{0}\u0002 to the edge of a cliff and throw them off. The cliff face is now covered in chunks of dead [redacted].",
```

Comment:
[redacted] looks weird here, maybe replace it with something else like 'villager' or 'their dead comrade' or something.